lib/govuk_index/elasticsearch_presenter.rb

```diff
@@ -0,0 +1,23 @@
+module GovukIndex
+  class ElasticsearchPresenter
+    def initialize(payload)
+      @payload = payload
+    end
+
+    def call
+      raise Indexer::ValidationError.new("base_path missing from payload") if payload["base_path"].nil?
```

Comment:
Just a thought on increasing the validations on this class. It might be worth looking at using the ActiveModel validations code when we extend the checks we are doing (I am not suggesting changing this now)